OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -415,6 +417,10 @@ public static void LoadActors(IniFile file, string section, List<string> players
 
 		public abstract string ParseTreeActor(string input);
 
+		public virtual int ModifyLocationX(string input) { return 0; }
```

Comment:
Alternatively you could pull out a `ParseActor` (like `ParseTreeActor` below) and then move the `LocationInit` stuff into both of these.